python/ray/_private/ray_constants.py

```diff
@@ -39,6 +39,9 @@ def env_set_by_user(key):
 # Whether event logging to driver is enabled. Set to 0 to disable.
 AUTOSCALER_EVENTS = env_integer("RAY_SCHEDULER_EVENTS", 1)
 
+# Filter level under which events will be filtered out, i.e. not printing to driver
+RAY_LOG_TO_DRIVER_EVENT_LEVEL = os.environ.get("RAY_LOG_TO_DRIVER_EVENT_LEVEL", "INFO")
```

Comment:
Can we do WARNING by default? Or does it break existing legacy autoscaler logs?

Reply:
legacy is INFO.

Reply:
but we could also change the leagcy level.